tests/data/PG3/shared/autoreduce/reduce_PG3.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
+import numpy as np
+import sys
+import time
+from datetime import datetime
+
+print("Running reduction for " + sys.argv[1] + " at " + datetime.isoformat(datetime.now()))
+
+_ = np.random.rand(100000, 10000)
```

Comment:
Please add a comment about this intentionally taking up a bunch of memory. It'll help later when someone stumbles across it.